index.html

```diff
@@ -0,0 +1,169 @@
+            </p>
+            <ul>
+                <li class="list">HTML</li>
+                <li class="list">CSS</li>
+                <li class="list">JAVASCRIPT</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+        </div>
+        <div class="cards">
+            <img src="images/card2.png" alt="">
+            <h1 class="card-title">Multi-Post Stories</h1>
+            <div class="card-text">
+                <h6>CANOPY</h6>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p class="para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul>
+                <li class="list">HTML</li>
+                <li class="list">CSS</li>
+                <li class="list">JAVASCRIPT</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+        </div>
+        <div class="cards">
+            <img src="images/card3.png" alt="">
+            <h1 class="card-title">Multi-Post Stories</h1>
+            <div class="card-text">
+                <h6>CANOPY</h6>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p class="para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul>
+                <li class="list">HTML</li>
+                <li class="list">CSS</li>
+                <li class="list">JAVASCRIPT</li>
+            </ul>
+            <button class="btn">See Project</button>
+        </div>
+
+        <div class="cards">
+            <img src="images/card4.png" alt="">
+            <h1 class="card-title">Multi-Post Stories</h1>
+            <div class="card-text">
+                <h6>CANOPY</h6>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p class="para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul>
+                <li class="list">HTML</li>
+                <li class="list">CSS</li>
+                <li class="list">JAVASCRIPT</li>
+            </ul>
+            <button class="btn">See Project</button>
+        </div>
+    </div>
+
+    <!-- Myself-->
+    <section class="myself">
+        <h1>About Myself</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+            my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+            contact me.</p>
+        <p>LET’S CONNECT</p>
+        <div class="connect">
```

Comment:
- [ ] For good HTML semantics and as required for the project make sure you use a `hx(e.g.: h2, h3,...)` tag on **"LET’S CONNEC..."** text instead of `<p>` tag as it is sub-title and not a paragraph.